### PR TITLE
Temporarily log to Honeybadger when a drupal link is loaded

### PR DESCRIPTION
**Pull request by joshlory**
Pegasus sets drupal.code.org as the base folder for code.org in [sites.v3/code.org/config.json](https://github.com/code-dot-org/code-dot-org/blob/7564066231e74aaeee763becfb3af70a84ce5dc5/pegasus/sites.v3/code.org/config.json#L2).  This means URIs that aren't found in `sites.v3/code.org` fall through to `sites.v3/drupal.code.org`.

Track which URIs fall through to sites.v3/drupal.code.org, so these assets can be moved and the old site folder removed.

screen shot 2015-12-11 at 2 20 50 pm

screen shot 2015-12-11 at 2 24 43 pm

### CODE REVIEW

File: pegasus/router.rb [hunk 1]
```diff
@@ -406,13 +406,24 @@ def post_process_html_from_markdown(full_document)
       full_document
     end
 
+    def log_drupal_link(dir, uri)
+      if dir == 'drupal.code.org'
+        Honeybadger.notify(
+          error_class: "Link to v3.sites/drupal.code.org",
+          error_message: "#{uri} fell through to the base config directory"
+        )
```
Comment (laurelfan) [inline pegasus/router.rb:416]: it looks like you could add something like `environment_name: drupal_#{Rack.env}` to the options here?

http://www.rubydoc.info/github/honeybadger-io/honeybadger-ruby/Honeybadger.notify

**Comment by laurelfan** [comment]
while you are in honeybadger pegasus maybe we should be setting the environment in the Honeybadger.config (even though we only load honeybadger in prod for pegasus now) here
https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/router.rb#L96

File: pegasus/router.rb [hunk 2]
```diff
@@ -406,13 +406,24 @@ def post_process_html_from_markdown(full_document)
...
+    end
 
     def resolve_static(subdir, uri)
       return nil if settings.non_static_extnames.include?(File.extname(uri))
 
       @dirs.each do |dir|
         path = content_dir(dir, subdir, uri)
-        return path if File.file?(path)
+        if File.file?(path)
+          log_drupal_link(dir, uri)
```
Comment (laurelfan) [inline pegasus/router.rb]: would including the path in the log message make it easier to find/fix?

**Comment by joshlory** [comment]
Added env and path context.  Good idea to add environments to Pegasus Honeybadger, but I think it should be a separate PR.

**Comment by laurelfan** [comment]
One more -- do you have access to the referer? that might help track these down too.

Otherwise LGTM

**Comment by joshlory** [comment]
Yeah, referrer would be great to have but it gets stripped by CloudFront :(

**Comment by laurelfan** [comment]
of course it does... :cloud: 

LGTM

**Pull request merged**